src/ssh_helpers/main.go

```diff
@@ -30,7 +32,9 @@ func StartRemoteNvim(nvrhContext context.NvrhContext) {
 		fmt.Sprintf("$SHELL -i -c '%s'", nvimCommand),
 	)
 
-	// sshCommand.Stdout = os.Stdout
+	if runtime.GOOS == "windows" {
+		sshCommand.Stdout = os.Stdout
+	}
 	// sshCommand.Stderr = os.Stderr
```

Comment:
Setting either `.Stdout` or `.Stderr` worked, I'm going with stdout so no strange errors pop up and potentially worry users.